charts/kyverno-aws-adapter/values.yaml

```diff
@@ -41,3 +35,5 @@ image:
   pullPolicy: Always
```

Comment:
Do we really want `Always` here ?
cc @anusha94 @pns-nirmata

Reply:
'Always' allows pulling a new version of image corresponding to a tag in case it is updated. Say, xyz:v0.1.0 is upgraded to xyz:v0.1.1, then having an image with xyz:v0.1 can get the fixes done in v0.1.1 by just a restart. No change to the gitops yaml is needed. Secondly, fail fast if the image is erroneous, needs to be removed.

Downside is of course, it takes time.

Reply:
I don't have a better answer. But let's change it to `IfNotPresent` since that's the default behavior usually. The user can always provide a different value if needed.

Reply:
Yes, that can be the default.

Reply:
Leaves an option. Like ":latest". It is a pointer that could point to different versions.